Dockerfile.openssl

```diff
@@ -0,0 +1,80 @@
+FROM clearlinux:base as builder
+
+ARG QAT_DRIVER_RELEASE="qat1.7.l.4.9.0-00008"
+ARG QAT_ENGINE_VERSION="v0.5.44"
+ARG QAT_ENGINE_FLAGS="--disable-multibuff-offload"
+
+RUN swupd update && \
+    swupd bundle-add \
+    --skip-diskspace-check \
+    --allow-insecure-http \
+    devpkg-systemd \
+    devpkg-openssl \
+    c-basic \
+    curl \
+    git \
+    llvm9 \
+    python3-basic \
+    os-core-dev && \
+    git clone -b $QAT_ENGINE_VERSION https://github.com/intel/QAT_Engine && \
+    git clone https://github.com/intel/ipp-crypto && \
+    mkdir QAT_Lib && cd QAT_Lib && \
+    curl -L https://01.org/sites/default/files/downloads/$QAT_DRIVER_RELEASE.tar.gz | tar zx
+
+WORKDIR /QAT_Lib
+
+RUN sed -i -e 's/cmn_ko$//' -e 's/lac_kernel$//' quickassist/Makefile && \
+    KERNEL_SOURCE_ROOT=/tmp ./configure && \
+    make quickassist-all adf-ctl-all && \
+    install -m 755 build/libqat_s.so /usr/lib/ && \
+    install -m 755 build/libusdm_drv_s.so /usr/lib/ && \
+    install -m 755 build/adf_ctl /usr/bin/ && \
+    cd /ipp-crypto/sources/ippcp/crypto_mb && \
+    sed -i -e '/OPENSSL_USE_STATIC_LIBS/d' CMakeLists.txt && \
+    cmake . -B"../build" \
+    -DOPENSSL_INCLUDE_DIR=/usr/include/openssl \
+    -DOPENSSL_LIBRARIES=/usr/lib64 \
+    -DOPENSSL_ROOT_DIR=/usr/bin/openssl && \
+    cd ../build && \
+    make crypto_mb && make install && \
+    install -m 755 bin/vfy_ifma_cp_rsa_mb /usr/bin/ && \
+    install -m 755 bin/vfy_ifma_rsa_mb /usr/bin/ && \
+    cd /QAT_Engine && \
+    ./autogen.sh && \
+    ./configure \
+    $QAT_ENGINE_FLAGS \
+    --with-qat_dir=/QAT_Lib \
+    --enable-qat_debug \
```

Comment:
Is there a performance penalty related to debug options? Should this be also coming from environment?

Reply:
I forgot this, it should not be here.

Reply:
fixed